### PR TITLE
feat(visualizer): add icons for bicycles and busses in WebVisualizer

### PR DESCRIPTION
**Pull request by hoelger**
## Description

In the webVisualizer all vehicles were shown as cars, also if they are bicycles or buses. This PR fixes this, plus minor cleanup.

## Issue(s) related to this PR

* Resolves internal issue 893
  
## Affected parts of the online documentation

Only changes in the README, which are here in this PR.

## Definition of Done

**Prerequisites**

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] Your GitHub user id is linked with your Eclipse Account.

**Required**

- [x] The title of this merge request follows the scheme `type(scope): description`  (in the style of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary))
- [x] You have assigned a suitable label to this pull request (e.g., `enhancement`, or `bugfix`)
- [x] `origin/main` has been merged into your Fork.
- [x] Coding guidelines have been followed (see CONTRIBUTING.md).
- [x] All checks on GitHub pass.
- [x] All tests on [Jenkins](https://ci.eclipse.org/mosaic/job/mosaic/) pass.

**Requested** (can be enforced by maintainers)

- [ ] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [ ] If a bug has been fixed, a new unit test has been written (beforehand) to prove misbehavior
- [ ] There are no new SpotBugs warnings.

### CODE REVIEW

File: fed/mosaic-output/src/main/resources/web/visualizer-dev.js [hunk 1]
```diff
@@ -11,279 +11,304 @@ import {Icon, Style} from 'ol/style'
...
-
-    // Update style
-    const style = this.createStyle()
-    this.marker.setStyle(style)
-
-    if ((Date.now() - this.timeStateChange) > 500) {
-      // Clear sending/receiving states
-      this.state.sending = false
-      this.state.receiving =  false
-      this.timeStateChange = 0
```
Comment (kschrab) [inline fed/mosaic-output/src/main/resources/web/visualizer-dev.js:166]: Why did you remove this?

Comment (hoelger) [reply fed/mosaic-output/src/main/resources/web/visualizer-dev.js:166]: The line has no effect, it will again run into this if block after the change. also mentioned this briefly in the commit msg

File: fed/mosaic-output/src/main/resources/web/visualizer.js
Comment (kschrab) [inline fed/mosaic-output/src/main/resources/web/visualizer.js:1]: This file is soo big suddently.. what happened here?
